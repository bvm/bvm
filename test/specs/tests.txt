# these don't currently work... I've just been manually testing these
deno eval "console.log(""hello"")"
deno eval "console.log(2 != 3)"
# doesn't work on ps
deno eval -p JSON.stringify({})
# ps requires quotes
deno eval -p "JSON.stringify({})"
deno eval "console.log(Deno.args[0])" lib=""
deno eval "console.log(Deno.args[0])" -- lib=test,other
