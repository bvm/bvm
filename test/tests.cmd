REM This was failing due to quotes
deno eval "console.log(""hello"")"

REM todo, add tests for something like lib="" and ensure the equals sign is preserved