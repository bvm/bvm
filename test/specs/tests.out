hello
0
true
0
{}
0
lib=
0
lib=test,other
0
